state_synchronizer/src/coordinator.rs

```diff
@@ -64,6 +64,7 @@ pub(crate) struct SyncCoordinator<T> {
     // value format is (expiration_time, known_version, limit)
     subscriptions: HashMap<PeerId, (SystemTime, u64, u64)>,
     executor_proxy: T,
+    last_requested_peer_id: Option<PeerId>,
```

Comment:
can we move it inside peer_manager. You can imagine later it will be replaced by other DS that tracks all peers and corresponding requests